src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx

```diff
@@ -60,28 +60,28 @@ export class NotificationConsoleComponent extends React.Component<
     { key: "Error", text: "Error" },
   ];
   private headerTimeoutId?: number;
-  private prevHeaderStatus: string | null;
+  private prevHeaderStatus: string;
   private consoleHeaderElement?: HTMLElement;
 
   constructor(props: NotificationConsoleComponentProps) {
     super(props);
     this.state = {
-      headerStatus: "",
-      selectedFilter: NotificationConsoleComponent.FilterOptions[0].key || "",
-      isExpanded: props.isConsoleExpanded,
+      headerStatus: undefined,
+      selectedFilter: NotificationConsoleComponent.FilterOptions[0].key,
+      allConsoleData: props.consoleData ? [props.consoleData] : [],
```

Comment:
As I mentioned in the previous comment, this is still needed. Everytime a new `consoleData` is passed from Explorer, it will be added to the `allConsoleData` list and triggers a re-render.